actioncable/lib/action_cable/server/base.rb

```diff
@@ -54,15 +54,15 @@ def event_loop
 
       # The worker pool is where we run connection callbacks and channel actions. We do as little as possible on the server's main thread.
       # The worker pool is an executor service that's backed by a pool of threads working from a task queue. The thread pool size maxes out
-      # at 4 worker threads by default. Tune the size yourself with config.action_cable.worker_pool_size.
+      # at 4 worker threads by default. Tune the size yourself with the `config.action_cable.worker_pool_size` configuration option.
```

Comment:
I think it's better to explicitly specify that this is a configuration option -- kind of confusing what it's otherwise used for IMO -- just kind of a random code snippet :)